scripts/check_frontend_test_coverage.py

```diff
@@ -342,6 +342,7 @@
     'topic-summary-tile.directive.ts',
     'TopicObjectFactory.ts',
     'topics-and-skills-dashboard-backend-api.service.ts',
+    'topics-list.component.ts',
```

Comment:
Why was this added to the list?

Reply:
not fully covered

Reply:
Can we cover it fully? It seems to be fully covered before, so lets not add to the list

Reply:
Done